Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:1.23.2
+
+WORKDIR /app
+
+COPY . .
+
+RUN go build -o concom main.go
```

Comment:
Let's try to use a multi-stage here to copy the binary out to a distroless image.

Reply:
https://github.com/compliance-framework/configuration-service/blob/main/Dockerfile